### ISSUE TITLE
Reordering tabs still crashes when running elevated

### ISSUE THREAD

**Issue by DJackman123** [microsoft/terminal#5564]
# Environment

```none
Windows build number: 10.0.19041.0
Windows Terminal version: 0.11.1133.0

Any other software?
```

# Steps to reproduce

Launch Windows Terminal as administrator.  Create a couple of tabs.  Use the mouse to click and drag one of the tabs.

# Expected behavior

Nothing should happen

# Actual behavior

Windows Terminal crashes

#4874 was supposed to prevent tab reordering while elevated and it either doesn't work (can't remember if I tested this when it first released) or some later change broke it again.

**Comment by Sandra2001** [comment]
**My Windows terminal does not crash.**
Your @Sandra2001

**Comment by DJackman123** [comment]
Just to double-check: you are running Terminal as administrator, correct?

**Comment by DHowett-MSFT** [comment]
Sorry about that. We regressed this in an error handling cleanup pass. Fix is in #5542.
